x-pack/plugins/maps/public/connected_components/right_side_controls/layer_control/layer_toc/toc_entry/toc_entry_actions_popover/toc_entry_actions_popover.tsx

```diff
@@ -36,7 +36,7 @@ export interface Props {
   showThisLayerOnly: (layerId: string) => void;
   supportsFitToBounds: boolean;
   toggleVisible: (layerId: string) => void;
-  editModeActiveForLayer: boolean;
+  isFeatureEditorOpenForLayer: boolean;
```

Comment:
Can you remove `isFeatureEditorOpenForLayer` from props. Its not used anywhere in the file